.buildkite/scripts/steps/cloud/deploy.json

```diff
@@ -201,7 +201,8 @@
           ],
           "kibana": {
             "version": null,
-            "docker_image": null
+            "docker_image": null,
+            "config": "config/theming.yml"
```

Comment:
TIL/FMI: what does that do exactly?

Reply:
Sorry,only seeing this now. It doesn't do anything, I was trying to figure out how to deploy the PR to Cloud with a config file set as a parameter.